doma-kotlin/src/main/kotlin/org/seasar/doma/kotlin/jdbc/criteria/declaration/KSetDeclaration.kt

```diff
@@ -5,7 +5,7 @@ import org.seasar.doma.jdbc.criteria.metamodel.PropertyMetamodel
 
 class KSetDeclaration(private val declaration: SetDeclaration) {
 
-    fun <PROPERTY> value(left: PropertyMetamodel<PROPERTY>, right: PROPERTY?) {
+    fun <PROPERTY> value(left: PropertyMetamodel<PROPERTY>, right: PROPERTY) {
```

Comment:
We might be better off doing it this way.

```kotlin
fun <PROPERTY: Any> value(left: PropertyMetamodel<PROPERTY>, right: PROPERTY?) {
```

What do you think?

Reply:
@momosetkn Could you please share your opinion?
This is related #1143

Reply:
I want set null only by `PropertyMetamodel<String?>` until a while ago.

But `PropertyMetamodel<String?>` does not actually exist.( but Unless it was done intentionally.)
I will fix to according to the not-null generics principles.